packages/error-boundary-fallback/src/components/ErrorBoundary/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
```

Comment:
why are we not using https://github.com/getsentry/sentry-javascript/blob/master/packages/react/src/errorboundary.tsx
thought I had seen this in this pr sometime back. @HaseemUlHaq

Reply:
sorry, meant this: https://www.npmjs.com/package/@sentry/react

Reply:
@p-netm I have changed earlier implementation where I was using Fallback prop from Sentry, which was not testable in development mode.
Now we are using React Error Boundary, and throwing error to Sentry after catching issue which is more effective and recommended one.

Reply:
@p-netm We are still using the same package `@sentry/react` for Sentry.

Reply:
Sentry also provisions an Error Boundary, is that what we are using, my understanding was this was going to be a fallback component for Sentry's error boundary, but it seems to be an Error boundary

Reply:
@p-netm 
Sentry also uses the Error boundary to catch errors, so the concept behind both is same.
The implementation is done in such a way that if user wants to consume our package and only want to use Error Boundary without sentry, he can use that. and if he want to use Sentry with this, then user just have to pass the DSN.

Reply:
my concern is that:
1. we are duplicating effort
2. The Sentry solution is probably more robust, a question here is if this component will report errors with the same comprehensivity as that provisioned by sentry
3. Seems like sentry functionality is hardcoded into this component so I disagree with the statement that *if user wants to consume our package and only want to use Error Boundary without sentry, he can use that.*

Reply:
- Regarding implementation and flow, Both uses Fallback component. So effort remains same for both.
- Sentry Uses React Error Boundary in its fallback component to catch errors, and Fallback component is what we create and pass via props.
- Current implementation is also the same, Using Error boundary and then throwing error to sentry. Effectiveness for both remains same.